docs/team/sebastiantoh.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Sebastian Toh's Project Portfolio Page
+---
+
+## Project: StonksBook (WIP)
+
+AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
Should change to StonksBook here as well.

Reply:
Thanks for pointing that out!